conda-store-server/Dockerfile

```diff
@@ -23,8 +23,7 @@ RUN cd /opt/conda-store-server && \
 ENV PATH=/opt/conda/condabin:/opt/conda/envs/conda-store-server/bin:/opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:${PATH}
 ENV TZ=America/New_York
 
-RUN chown -R 1000:1000 /opt/conda/pkgs && \
```

Comment:
Is this recursive command really needed here for something?. I found that removing the recursive `chown` of `/opt/conda/pkgs` speeds up image builds by literally ~60X (from 80s to 1.something), and doesn't seem to break the operation of the resulting `conda-store` dev build.